interceptors/Mementifier.cfc

```diff
@@ -121,9 +121,47 @@ component{
 				thisName = ( md.keyExists( "entityName" ) ? md.entityName : listLast( md.name, "." ) );
 			}
 
-			thisMemento.defaultIncludes = ormGetSessionFactory()
-				.getClassMetaData( thisName )
-				.getPropertyNames();
+			var ormUtil = server.coldfusion.productname == "ColdFusion Server" ? new cborm.models.util.CFORMUtil() : new cborm.models.util.LuceeORMUtil(); 
+			var classMd = ormUtil.getSessionFactory( ormUtil.getEntityDatasource( thisName ) ).getClassMetaData( thisName );
+			var typeMap = arrayReduce( 
+								classMd.getPropertyNames(), 
+								function( mdTypes, propertyName ){
+									var propertyType = classMd.getPropertyType( propertyName );
+									var propertyClassName = getMetadata( propertyType ).name;
+
+									mdTypes[ propertyName ] = propertyClassName;
+									return mdTypes;
+								}
+								,{});
+			
+			thisMemento.defaultIncludes = typeMap.keyArray().filter( function( propertyName ){
+					switch( listLast( typeMap[ propertyName ], "." ) ){
+						case "BagType":
+                    	case "OneToManyType":
+						case "ManyToManyType":
+						case "ManyToOneType":
+						case "OneToOneType":
+						case "BinaryType":{
+                          return false;
+                    	}
+						default:{
+						  return true;
+						}
+					}
+			} );
+
+			// Append primary keys
+
+			var hibernateMD = ormUtil.getEntityMetadata(
```

Comment:
if we are using cborm, why not just instantiate a BaseORM Service and call what you need from it?

Reply:
That couples it more to the BaseORM service, though that's a valid pushback.  Since mementifier doesn't have injection and BaseORMService injects wirebox, I would have to inject wirebox into Mementifier, and call it via `getInstance`.  

Using the utils directly was my attempt to keep it decoupled from injecting wirebox.

Reply:
Look above, you don't need to use WireBox. You can use the orm service by pure instantiation.